app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -24,11 +28,19 @@ export class OrderEntryPage extends React.Component {
       <div>
         <div />
         <PatientDashboard {...this.props} />
-        <SearchAndAddOrder
-          outpatientCareSetting={this.props.outpatientCareSetting}
-          inpatientCareSetting={this.props.inpatientCareSetting}
-          location={this.props.location}
-        />
+        { globalPropertyEncounterType.length > 0 && error === '' ?
+          <SearchAndAddOrder
+            outpatientCareSetting={this.props.outpatientCareSetting}
+            inpatientCareSetting={this.props.inpatientCareSetting}
+            location={this.props.location}
+          /> :
+
+          <div>
+            <p className="text-align-center error-notice">
+            Please contact admin to configure global property for encounterType.
```

Comment:
Shouldn't it be more user friendly to tell them what exactly they need to do? That is, tell them to create an encounter type with the expected name.